### DISCUSSION TITLE
How to merge new version?

### DISCUSSION THREAD

**Discussion by dipta007** [alshedivat/al-folio#2151] [Q&A] [answered]
How do you guys merge new version?
I was doing it manually as there were a lot of merge conflicts.
But is there any easier or automated way to do that?

**Comment by george-gca** [comment, marked as answer]
AFAIK unfortunately not. Git already does the best it can do automate the merge process, but in the end more complex merges must be done manually to ensure everything goes as expected. What I do in my case is try to keep my site as up-to-date as possible with the template. In my current setup I set [meld](https://meldmerge.org/) as my [mergetool](https://git-scm.com/docs/git-mergetool), and basically use it kind of like in [this video](https://www.youtube.com/watch?v=rBep9HWsOHY). The difference is that I set `meld` to be my default `mergetool`, with:

```bash
git config --global merge.tool meld
```

Then when solving merge I just call `git mergetool` and it automatically opens `meld`.

But in the past I had to clone the repo again and do a manual comparison between entire directories with [meld](https://meldmerge.org/).
